CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+2. Once done, [fork the meilisearch-kubernetes repository](https://help.github.com/en/github/getting-started-with-github/fork-a-repo) in your own GitHub account. Ask a maintainer if you want your issue to be checked before making a PR.
+3. [Create a new Git branch](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-and-deleting-branches-within-your-repository).
+4. Review the [Development Workflow](#workflow) section that describes the steps to maintain the repository.
+5. Make the changes on your branch.
+6. [Submit the branch as a PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork) pointing to the `main` branch of the main meilisearch-kubernetes repository. A maintainer should comment and/or review your Pull Request within a few days. Although depending on the circumstances, it may take longer.<br>
+ We do not enforce a naming convention for the PRs, but **please use something descriptive of your changes**, having in mind that the title of your PR will be automatically added to the next [release changelog](https://github.com/meilisearch/meilisearch-kubernetes/releases/).
+
+## Development Workflow
+
+### Setup <!-- omit in toc -->
+
+In order to use the different tools on this repository, you will first need to:
+
+- [Install Kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl) on your machine. This will allow you to run commands against your Kubernetes cluster with the `kubectl` command.
+
+- [Install Helm](https://helm.sh/docs/intro/install/). This will allow you to run the `helm` command, enabling the installation and manipulation of Helm charts.
+
+- A Kubernetes cluster up and running. Any Kubernetes cluster can be used, on the cloud or locally installed on your machine. This can be easily achieved by [installing minikube](https://minikube.sigs.k8s.io/docs/start/), and creating a local cluster by running:
+
+```bash
+minikube start
+```
+
+You can install and test the Helm chart by running:
+
+```bash
+helm install meilisearch-chart-test -f charts/meilisearch/values.yaml charts/meilisearch
+```
+
+An easy way to access your MeiliSearch instance and test that it is up and running is by using port-forwarding. This can be achieved by running:
+
+```bash
+kubectl port-forward --namespace default svc/meilisearch 7700:7700
+```
+
+Finally, to uninstall the chart, run:
+
+```bash
+helm uninstall meilisearch-chart-test 
+```
+
+### Tests and Linter <!-- omit in toc -->
+
+
+Each PR should pass the linter to be accepted.
+
+```bash
+# Linter
+helm lint charts/meilisearch 
+```
+
+Additionally, the GitHub CI will run a test to check if there are changes introduced to the charts. If changes were introduced, it will require you to update the Chart version 
```

Comment:
```suggestion
Additionally, the GitHub CI will run a test to check if there are changes introduced to the charts. If changes were introduced, it will require you to update the Chart version.
```